internal/querynode/query_collection.go

```diff
@@ -52,6 +52,13 @@ type queryMsg interface {
 	TimeoutTs() Timestamp
 }
 
+// queryCollection manages and executes the retrieve and search tasks, it can be created
+// by LoadCollection or LoadPartition, but can only be destroyed by ReleaseCollection.
+// Currently query behaviors are defined below, if:
+// 1. LoadCollection --> ReleaseCollection: Query would be failed in proxy because collection is not loaded;
+// 2. LoadCollection --> ReleasePartition: Not allowed, release would failed;
+// 3. LoadPartition --> ReleaseCollection: Query would be failed in proxy because collection is not loaded;
+// 4. LoadPartition --> ReleasePartition: Query in collection should return empty results, and query in partition should return notLoaded error.
```

Comment:
If there are no other partions in the querynode after the release partition, querycoord will directly call the release collection

Reply:
Is there any test case which is loadPartition and then releasePartition, does the result of this case is return empty result rather than throw exception.

Reply:
thers is no such case in ci,  nightly ci may has